sunpy/map/map_factory.py

```diff
@@ -260,13 +262,14 @@ def __call__(self, *args, **kwargs):
 
         # Hack to get around Python 2.x not backporting PEP 3102.
         composite = kwargs.pop('composite', False)
-        
+
         # MapCube Deprecation
         cube = kwargs.pop('cube', False)
         if cube:
-            warnings.warn('MapCube is now deprecated and renamed MapSequence. Please use the syntax Map(sequence=True) instead of Map(cube=True).',
-                          DeprecationWarning, stacklevel=2)
-            
+            warnings.warn('MapCube is now deprecated and renamed MapSequence.\
```

Comment:
don't use line continuation here, just break inside the bracket. i.e

```python
("athing "
 "another thing")
```

just make sure to remember the space at the end of the first string.